### PR TITLE
add to metadata: floor, lat, lon

### PR DESCRIPTION
**Pull request by noursaidi**

### CODE REVIEW

File: schema/model_system.json [hunk 1]
```diff
@@ -38,6 +43,20 @@
               "type": "number"
             }
           }
+        },
+        "coordinates": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "lat": {
+              "description": "The `lattitude` coordinate of the device in the WSG84 coordinate system",
```
Comment (pisuke) [inline schema/model_system.json:52]: typo: latitude

Comment (pisuke) [reply schema/model_system.json:52]: Also WGS84 (world geodetic system)

File: schema/model_system.json [hunk 2]
```diff
@@ -38,6 +43,20 @@
...
+              "type": "number"
+            },
+            "lon": {
+              "description": "The `longitude` coordinate of the device in the WSG84 coordinate system",
```
Comment (pisuke) [inline schema/model_system.json:56]: WGS84

File: schema/model_system.json [hunk 3]
```diff
@@ -38,6 +43,20 @@
...
+        },
+        "coordinates": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "lat": {
+              "description": "The `lattitude` coordinate of the device in the WSG84 coordinate system",
+              "type": "number"
+            },
+            "lon": {
```
Comment (pisuke) [inline schema/model_system.json:55]: let's use long

File: tests/schemas/metadata/example.json [hunk 4]
```diff
@@ -6,9 +6,15 @@
     "location": {
       "site": "US-SFO-XYY",
       "section": "NW-2F",
+      "floor": "2",
```
Comment (grafnu) [inline tests/schemas/metadata/example.json:9]: Add "room" in the example?

**Review by grafnu — APPROVED** [review]
Just one minor comment... but, go ahead and submit when you're ready.

**Pull request merged**